src/main/java/nl/tudelft/contextproject/script/Shot.java

```diff
@@ -0,0 +1,86 @@
+package main.java.nl.tudelft.contextproject.script;
+
+import main.java.nl.tudelft.contextproject.camera.Camera;
+import main.java.nl.tudelft.contextproject.presets.Preset;
+
+/**
+ * Class to represent a shot to be taken by a Camera.
+ *
+ * @author Etta Tabe Takang Kajikaw
+ * @since 0.2
+ */
+public class Shot {
+    private int number;
+    private Camera camera;
+    private Preset preset;
+
+
+    /**
+     * Creates a shot instance with each shot having a number,
+     * camera and preset.
+     * @param num is the shot number
+     * @param cam is the camera used to make the shot.
+     * @param pres is the preset used for the shot.
+     */
+    public Shot(int num, Camera cam, Preset pres) {
+        this.number = num;
+        this.camera = cam;
+        this.preset = pres;
+    }
+
+    /**
+     * Returns the number of the shot.
+     * @return the number of the shot.
```

Comment:
This javadoc is incorrectly formulated (same goes for the rest of the class). See https://google.github.io/styleguide/javaguide.html#s7.2-summary-fragment